jerry-core/parser/js/js-parser.c

```diff
@@ -1614,6 +1614,14 @@ parser_post_processing (parser_context_t *context_p) /**< context */
     }
 
     tagged_base_p[-1] = (ecma_value_t) context_p->tagged_template_literal_cp;
+
+    ecma_collection_t *collection_p = ECMA_GET_INTERNAL_VALUE_POINTER (ecma_collection_t,
```

Comment:
Would it make sense to have an assert to check if the `collection_p` is not a null value?

Reply:
IMO that a bit unnecessary, since `ECMA_GET_INTERNAL_VALUE_POINTER_ANY` may returns `NULL`, but `ECMA_GET_INTERNAL_VALUE_POINTER` never.

Reply:
Ahh.. true!